automated install/basic-install.sh

```diff
@@ -838,17 +838,13 @@ install_dependent_packages() {
   for i in "${argArray1[@]}"; do
     echo -n ":::    Checking for $i..."
     if ${PKG_MANAGER} -q list installed "${i}" &> /dev/null; then
-      echo " installed!"
+      echo " already installed!"
     else
-      echo " added to install list!"
-      installArray+=("${i}")
+      echo " installing!"
+      "${PKG_INSTALL[@]}" "${i}"
```

Comment:
There is a lot of overhead if you run the package manager for every package on its own.